index.rst

```diff
@@ -44,14 +44,636 @@
+That group was not focused on on-the-fly functionality but many of the findings were applicable and taken into account here when it made sense to do so.
+
+This report is structured into sections, where the first addresses each of the charge questions individually.
+This is followed by a section on findings that are pertinent to the commissioning team but fall outside the scope of the charge, and subsequently a section outlining the proof-of-concept demonstrators that helped inform our recommendations.
+
+
+Responses to Charge Questions and Deliverables
+==============================================
+
+The use-cases referenced throughout the document are all found on the :ref:`pg-D1-Use-cases` page.
+In each of the subsections
+
+.. _Deliverable 1:
+
+Deliverable 1: Use-Cases
+------------------------
+
+.. note:: 
+
+   The deliverable description from the `charge`_ is in the note below to ease readability 
+
+   1. A series of use-cases where feedback to observers is required. A reduced set of use-cases should be created as a regular reference during design and development.
+
+      - Use-cases should be complete, including which inputs are required (e.g. SAL Script, EFD, LFA, external source), desired manipulations, logic-based operations/calculations, and how the desired artefacts are presented to the user (e.g. display images and/or graphs).
+  
+A solicitation for use-cases was made to the Project stagg via the Commissioning meeting and a general call on the Slack channel.
+A meeting was held to define what the group was looking for and provided examples.
+Members were also encouraged to pass the call for information to whomever may be able to provide useful input.
+The responses from the community can be found `here <https://confluence.lsstcorp.org/display/LSSTCOM/2021-05-14+On-the-fly+analysis+for+observers+Meeting+Minutes#id-20210514OntheflyanalysisforobserversMeetingMinutes-On-the-flyAnalysisUse-Cases>`_.
+
+Many of the examples from the community had significant overlap of information.
+To amalgamate the information and create a basis from which to work, a series of standardized use-cases were developed that followed a common format and layout.
+A standardized summary was also created to ease comparisons between use-cases.
+
+The standardized use-cases associated with this deliverable are found at :ref:`pg-D1-Use-cases` page.
+
+.. _Deliverable 2:
+
+Deliverable 2: Requirements
+---------------------------
+
+.. note:: 
+
+   The deliverable description from the `charge`_ is in the note below to ease readability 
+
+   2. A set of requirements to augment, clarify, or remove ambiguity that may occur from looking only at the use-cases.
+      
+      - This is not a formal and all-encompassing LSE-type requirements document.
+
+Like the use-cases, the requirement deliverables have been separated into two different categories, those which are focused on tooling and procedures for on-the-fly analysis and image visualization.
```

Comment:
Should it be a ";" instead of a ","?

>Like the use-cases, the requirement deliverables have been separated into two different categories; ...

Reply:
Uhh, for a list you use a colon... but do two things count as a list? Google wasn't helpful but I modified it. 
A critical comma was missing though!